### PR TITLE
Roc 8054 ccbc changes

### PR DESCRIPTION
**Pull request by tribhuwan-hmcts**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-8052
https://tools.hmcts.net/jira/browse/ROC-8054

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/main/app/claims/models/claim.ts [hunk 1]
```diff
@@ -81,6 +81,7 @@ export class Claim {
   paperResponse: YesNoOption
   claimDocuments?: ClaimDocument[]
   proceedOfflineReason: string
+  handedOffToCcbcAt: Moment
```
Comment (daveoh) [inline src/main/app/claims/models/claim.ts:84]: Is this needed. I don't see it used anywhere?

Comment (tribhuwan-hmcts) [reply src/main/app/claims/models/claim.ts:84]: I initially thought date might be needed in history page, looks like date is not needed in that page

Comment (daveoh) [reply src/main/app/claims/models/claim.ts:84]: This should be removed then

Comment (tribhuwan-hmcts) [reply src/main/app/claims/models/claim.ts]: done

File: src/main/app/claims/models/claim.ts [hunk 2]
```diff
@@ -245,6 +245,8 @@ export class Claim {
       return ClaimStatus.CLAIMANT_RESPONSE_SUBMITTED
     } else if (this.moreTimeRequested) {
       return ClaimStatus.MORE_TIME_REQUESTED
+    } else if (this.state === 'BUSINESS_QUEUE') {
+      return ClaimStatus.BUSINESS_QUEUE
```
Comment (daveoh) [inline src/main/app/claims/models/claim.ts:249]: Can you add a test for this status. Similar to how we test for `TRANSFERRED`

File: src/main/app/claims/models/claim.ts [hunk 3]
```diff
@@ -467,6 +469,10 @@ export class Claim {
       return false
     }
 
+    if (this.state === 'BUSINESS_QUEUE') {
```
Comment (daveoh) [inline src/main/app/claims/models/claim.ts:472]: I would follow the pattern that we set and make this a method. See the isTransferred() method

File: src/test/app/claims/models/claim.ts [hunk 4]
```diff
@@ -1063,6 +1063,19 @@ describe('Claim', () => {
     })
   })
 
+  describe('handoffToCCBC', () => {
```
Comment (prudhvi-v) [inline src/test/app/claims/models/claim.ts:1066]: It is possible to refactor this test case? We cannot be testing the value we assigned

Comment (tribhuwan-hmcts) [reply src/test/app/claims/models/claim.ts:1077]: done

File: src/main/app/claims/models/claim.ts (hunk 3 above)
Comment (tribhuwan-hmcts) [reply src/main/app/claims/models/claim.ts]: done

**Pull request merged**